.github/workflows/upstream_testing.yml

```diff
@@ -10,4 +10,4 @@ jobs:
     uses: "nautobot/nautobot/.github/workflows/plugin_upstream_testing_base.yml@develop"
     with:  # Below could potentially be collapsed into a single argument if a concrete relationship between both is enforced
       invoke_context_name: "NAUTOBOT_SECRETS_PROVIDERS"
-      plugin_name: "nautobot-plugin-secrets-providers"
```

Comment:
I guess we need to update the upstream workflow before we can change the input variable to `app_name`.